datacite-data/build.gradle

```diff
@@ -33,6 +33,10 @@ task fetchSchema() {
     }
 }
 
+jar {
+    from "$buildDir/classes/java/main"
```

Comment:
is this for making `gradle clean build` work?

Reply:
No, this is for putting generated classes into the jar. I haven't been able to make a gradle clean build work yet, but I set a custom build command using jitpack.yml config.